src/ui/RosterItem.ts

```diff
@@ -29,6 +43,21 @@ export default class RosterItem {
       this.element.attr('data-subscription', this.contact.getSubscription());
       this.element.attr('data-date', this.contact.getLastMessageDate()?.toISOString());
 
+      let groups = this.contact.getGroups();
+
+      if (Client.getOption(SHOW_TAGS,false)&&groups&&groups.length>0)
+      {
+          this.element.attr('data-groups', groups.join(';'));
```

Comment:
Is the attribute actually used somewhere?

Reply:
now a search over files dit not find it anywhere